getting-started/installation-ami.md

```diff
@@ -0,0 +1,23 @@
+## Installing from an Amazon AMI (Ubuntu)
+
+TimescaleDB is currently available as an Ubuntu 16.04 Amazon EBS-backed AMI. AMIs are
+distributed by region, and our AMI is currently available in `us-east-1`, `us-east-2`,
+`us-west-1`, and `us-west-2`. Note that this image is built to use an EBS attached volume
+rather than the default disk that comes with EC2 instances.
+
+See below for the image id corresponding to each region for the most recent TimescaleDB version:
+
+Region | Image id
+--- | ---
+us-east-1 | ami-0ecf0cc105d5a8fc0
```

Comment:
on the top right in the AWS console when you select regions, it doesn't tell me if it's us-east-1 or us-east-2. It says N.Virginia. Should we include the associated region in plain English here?